_posts/2017-02-28-book-review-introduction-to-the-commandline.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+categories: book review
+title: Book review 'Introduction to the Command Line'
+tag: books linux programming
+---
+![introduction_to_command_line]({{ site.url
+}}/assets/images/book_image_introduction_to_the_commandline.jpg)
+
+
+## tl;dr
+
+
+Every chapter will introduce a bunch of comandline utilities and will point to
+its respective documentation for further learning. You should expect chapters
+describing from the [grep](https://www.gnu.org/software/grep/manual/grep.html)
+command to [GNU Octave](https://www.gnu.org/software/octave/) which is a
+scientific programming language. The chapters are independent from each other.
+The book is must read if you new to the
+[GNU/Linux](https://en.wikipedia.org/wiki/Linux) command line. If you new are at
+the intermediate level, then too investing time in reading this book will unveil
+a few surprises for you.
+
+
+## Detailed review
+
+The book is community driven and published under [FLOSS
+Manual](http://flossmanuals.net). It is a collaborative effort of the
+[FSF](http://www.fsf.org/) community. The fun part is you can contribute to this
+book by adding new chapters or improve an existing one. I fixed one typo to this
+book after reading. The best introduction is crafted comparing GUI based image
+editing tools with the most unknown command
+[convert](https://linux.die.net/man/1/convert). It conveys the importance of
+command line well to the reader. Initial chapters will present the overview of
+various [GNU/bash](https://www.gnu.org/software/bash/) commands. From my
+personal experience, you have to encounter them daily. The chapter of Command
+history shortcuts depicts geeky shell patterns. I will advise to not skip that
+chapter and read through once. The advanced session was not much advance for me.
```

Comment:
May be what you mean (and I find relevant) is `section` instead of `session`.

The advanced section was not so advance for me.